commodore/gitrepo.py

```diff
@@ -539,6 +539,33 @@ def _check_conflicts(self):
                 if stage != 0:
                     raise MergeConflict(path)
 
+    def _compute_changed_files(self) -> tuple[list[str], list[str]]:
+        """Return a list of files to add to the index and a list of files to remove from
+        the index."""
+        # We always want to stage untracked files. The implementation for
+        # `untracked_files` respects the repo's `.gitignore`.
+        to_add = self._repo.untracked_files
+        # We don't want to remove anything by default.
+        to_remove = []
+        # Stage deletions and add changes to `to_add`
+        changes = self._repo.index.diff(None)
+        if changes:
+            for c in changes.iter_change_type("D"):
```

Comment:
Does the library provide constants for the change type? If so, then I would use them instead.

Reply:
There's no constants for the change types, otherwise the code would probably use them already.

Reply:
I thought so, but you never know.